packages/agent/src/tool/read-file/ReadFileTool.ts

```diff
@@ -50,12 +50,22 @@ export const executeReadFile =
     workspacePath: string;
   }): ExecuteToolFunction<ReadFileInput, ReadFileOutput> =>
   async ({ input: { filePath } }) => {
-    // TODO try-catch
-    const fullPath = path.join(workspacePath, filePath);
-    const content = await fs.readFile(fullPath, "utf-8");
+    try {
+      const fullPath = path.join(workspacePath, filePath);
+      const content = await fs.readFile(fullPath, "utf-8");
 
-    return {
-      summary: `Read file ${filePath}`,
-      output: { content },
-    };
+      return {
+        summary: `Read file ${filePath}`,
+        output: { content },
+      };
+    } catch (error: any) {
+      console.log(error);
```

Comment:
This is an expected situation, so I'd prefer to remove the console.log

Reply:
certainly, this is a WIP at the moment :) it will be gone.